root/ngtemplate/sample/panels/isla-worksheet.tt

```diff
@@ -0,0 +1,13 @@
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title">
+            ISLA Worksheet
+        </h3>
+    </div>
+
+    <div class="panel-body">
+        <form action="[% c.uri_for_action('/sample/isla_worksheet', [ sample.id ]) %]" method="get">
```

Comment:
This can just be a link, there's no form parameters and I don't know if button styling is a necessary UI affordance here. That said if we add the option of prefilling F. lab PCR codes we'll need an input field.

Reply:
I was thinking more in terms of, what happens the next time Google decides it's a good idea to prefetch all links for speed? Although I guess there's no side-effects and the overhead of formatting an extraneous Excel file isn't that big a deal for us.